docker/cygnus-ngsi/Dockerfile

```diff
@@ -65,9 +66,9 @@ RUN yum -y install tar nc which git java-1.6.0-openjdk-devel && \
 # Create Cygnus log folder
 RUN mkdir ${FLUME_HOME}/log
 
-# Clone fiware-cygnus
-RUN git clone ${GIT_URL_CYGNUS} ${CYGNUS_HOME} && \
-    cd ${CYGNUS_HOME} && \
+# Instead of re-cloning, copy the fiware-cygnus repo, move there and checkout the propper branch
+COPY . ${CYGNUS_HOME}
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/993/commits/6fb3e11aafbe8610cb91c43b586a755004e06706. Finally, we decided to rollback to the re-cloning, in order to release a new version of Cygnus, and make some research on the COPY option during the new fresh sprint.